scripts/globals/abilities/pets/radial_arcana.lua

```diff
@@ -0,0 +1,45 @@
+-----------------------------------
+-- Ability: Radial Arcana
+-- Causes your luopan to vanish and restores MP of party members within area of effect.
+-----------------------------------
+require("scripts/globals/mobskills")
+require("scripts/settings/main")
+require("scripts/globals/status")
+require("scripts/globals/msg")
+-----------------------------------
+local ability_object = {}
+
+ability_object.onAbilityCheck = function(player, target, ability)
+    return 0, 0
+end
+
+ability_object.onPetAbility = function(target, pet, skill)
+    local master             = pet:getMaster()
+    local mpAmount  = math.floor(3 * pet:getMainLvl())
+    local mpRestore = mpAmount
```

Comment:
Alignment is a bit off here

Reply:
This is the bane of my life since using VS code, it loves to put nice little labels next to variables so you can see the alignments correctly, 
like this:
local id `:interger` = blah
fun times